app/(manager)/inventory.tsx

```diff
@@ -14,7 +15,7 @@ export default function InventoryManagementScreen() {
   useEffect(() => {
     const fetchData = async () => {
       try {
-        const user = JSON.parse(sessionStorage.getItem('user') || '{}');
+        const user = JSON.parse(await AsyncStorage.getItem('user') || '{}');
```

Comment:
this can't have sensitive data, the user id is something that we use for the fetching of the data, we need a azure service to handle this auth not  use the session storage

Reply:
i replace with import * as SecureStore from 'expo-secure-store';